modelci/finetuner/computer_vision_fine_tuning.py

```diff
@@ -38,22 +31,25 @@
 
 Note:
     See: https://pytorch.org/tutorials/beginner/transfer_learning_tutorial.html
+
+Reference:
+    https://github.com/PyTorchLightning/pytorch-lightning/blob/master/pl_examples/domain_templates/computer_vision_fine_tuning.py
 """
 
 import argparse
 from pathlib import Path
 from tempfile import TemporaryDirectory
-from typing import Generator, Optional, Union
+from typing import Generator, Optional, Callable
 
 import pytorch_lightning as pl
 import torch
-import torch.nn.functional as F
-from pl_examples import cli_lightning_logo
 from torch import optim
 from torch.nn import Module
-from torch.optim.lr_scheduler import MultiStepLR
+from torch.optim.lr_scheduler import StepLR
```

Comment:
why change to StepLR?

Reply:
PyTorch official fine-tune example use StepLR, as this is the simplest one.